src/file_store.js

```diff
@@ -13,51 +13,47 @@ const dirname = 'flex-integration-sdk';
 //
 // This is also what Flex CLI does, so for consistency we'll also
 // use the XDG base dir here.
-const configDir = os.platform() === 'darwin'
-      ? path.join(xdgBasedir.config, dirname)
-      : envPaths(dirname).config;
+const configDir =
+  os.platform() === 'darwin' ? path.join(xdgBasedir.config, dirname) : envPaths(dirname).config;
```

Comment:
Did `prettier` really put this on single line? All is fine, just wondering.

Reply:
Seems so. It definitely didn't like the old indentation.